### PR TITLE
feat(integrations): add support for marketo

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes
- Added configuration for marketo in providers.yaml
- Added marketo to docs-v2.
- Added marketo pages for Supported API groups in mint.json.
- Added marketo SVG file in template-logos folder.
- Consolidate marketo docs pages

## Issue ticket number and link
[EXT-68](https://linear.app/nango/issue/EXT-68/add-support-for-marketo-api)
## Test
The documentation update was reviewed using `npm run docs` and verifying on localhost.

### CODE REVIEW

File: docs-v2/integrations/all/marketo.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+-   [OAuth-related docs](https://developers.marketo.com/rest-api/authentication)
+-   [Marketo REST API docs](https://developers.marketo.com/rest-api)
+
+<Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
+
+## API gotchas
+
+-   Marketo offers client credentials token that authorizes as app instead of a user.
+-   Both `endpointURL` and `identityURL` can be found in the Admin > Web Services menu in the REST API section.
```
Comment (TBonnin) [inline docs-v2/integrations/all/marketo.mdx]: Can we add a direct link to this marketo admin page?

**Review by TBonnin — APPROVED** [review]

File: docs-v2/integrations/all/marketo.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+## Getting started
+
+-   [OAuth-related docs](https://developers.marketo.com/rest-api/authentication)
+-   [Marketo REST API docs](https://developers.marketo.com/rest-api)
+
+<Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
+
+## API gotchas
+
+-   Marketo offers client credentials token that authorizes as app instead of a user.
```
Comment (TBonnin) [inline docs-v2/integrations/all/marketo.mdx]: Not sure if this is super clear and/or why it is useful to know.

**Review by bodinsamuel — APPROVED** [review]

File: packages/shared/providers.yaml [hunk 3]
```diff
@@ -1328,6 +1328,15 @@ mailchimp:
     authorization_params:
         response_type: code
     docs: https://docs.nango.dev/integrations/all/mailchimp
+#Untested configuration. Please reach out if you have a test account that we can use to test it.
```
Comment (khaliqgant) [inline packages/shared/providers.yaml]: Agree with Samuel on another PR that this comment should be removed.

File: docs-v2/integrations/all/marketo.mdx (hunk 1 above)
Comment (hassan254-prog) [reply docs-v2/integrations/all/marketo.mdx]: Added reference to the base URL section since I couldn’t find a direct link in the provider’s documentation.

**Review by bodinsamuel — APPROVED** [review]

**Review by khaliqgant — APPROVED** [review]

**Pull request merged**